target/ppc/translate.c

```diff
@@ -6547,6 +6547,12 @@ static int times_16(DisasContext *ctx, int x)
     return x * 16;
 }
 
+static int64_t dw_compose_ea(DisasContext *ctx, int x)
+{
+    int64_t ea = 0xfffffffffffffe00 | (x << 3);
+    return ea;
```

Comment:
should be `return 0xfffffffffffffe00 | (x << 3);`

Reply:
Should it be `deposit64(0xfffffffffffffe00, 3, 6, x)`? Or maybe I didn't understand how deposit64 works

Thanks for the tip, I didn't know this deposit method exists.

Reply:
Yep, it's either `deposit64(0xfffffffffffffe00, 3, 6, x)` or `deposit64(0xfffffffffffffe00, 3, 9, x << 3)`, but the first seems more direct. You can also change the value to `0xfffffffffffffff8` if you prefer.